twirssi.pl

```diff
@@ -1935,44 +1936,44 @@ sub get_updates_child {
                       "ac:$username\n";
             }
         }
-        next if $error > $errors_beforehand;
+        next if scalar @error > $errors_beforehand;
```

Comment:
remove the word `scalar` - putting `@error` in a scalar context (next to the `>`) is sufficient